src/avfoundation.cs

```diff
@@ -9247,6 +9438,10 @@ enum AVCaptureDeviceType {
 		[iOS (10, 2)]
 		[Field ("AVCaptureDeviceTypeBuiltInDualCamera")]
 		BuiltInDualCamera,
+
+		[iOS (11, 1)]
```

Comment:
missing NoMac, NoTV and NoWatch

Reply:
Those decorators are already present in line 9422 on top of the enum, we have NoTV, NoMac, NoWatch.